bin/cylc-graph

```diff
@@ -204,6 +207,9 @@ def main():
 
     (options, args) = parser.parse_args()
 
+    if no_x:
+        sys.exit(1)
```

Comment:
Could we just `return`/`sys.exit(0)` if `--help` is used? Doesn't seem like it should be a non-zero exit code as its successfully printing help details...